src/test/resources/examples/resultset/valid-resultset-example.json

```diff
@@ -39,14 +39,24 @@
   "processFolders": [
     {
       "name": "salmon",
-      "path": "./",
+      "path": "./salmon",
       "children": [
         {
           "name": "salmon.merged.gene_tpm",
           "fileType": "tsv",
-          "path": "salmon/salmon.merged.gene_tpm"
+          "path": "./salmon/salmon.merged.gene_tpm"
         }
       ]
     }
-  ]
+  ],
+  "runID": {
+    "name": "run_id",
+    "fileType": "txt",
+    "path": "./run_id.txt"
+  },
+  "inputIDs": {
+    "name": "input_ids",
+    "fileType": "txt",
+    "path": "./input_ids.txt"
```

Comment:
So that's the file containing the Sample IDs for qPostman, right?
I would maybe suggest to call this one `sampleIDs` so that it is clear that it refers to the sample IDs (QBIC code).

Reply:
addressed in ec2d1dc